### PR TITLE
Adding Buffalo, NY

### PR DESCRIPTION
**Pull request by ceebs**
Referencing Issue #154

### CODE REVIEW

File: _emails/buffalo.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  - choyt@city-buffalo.com
+  - mnowakowski@city-buffalo.com
+  - rfontana@city-buffalo.com
+  - uwingo@city-buffalo.com
+  - darivera@city-buffalo.com
+  - jgolombek@city-buffalo.com
+  - cscanlon@city-buffalo.com
+  - rwyatt@city-buffalo.com
+subject: Buffalo, NY Resident for Defunding the BPD
+body: "My name is [your name], and I am a resident of Buffalo. The incident as reported by WBFO (NPR) on June 4, 2020 is appalling and inhumane. This is a display of egregious police brutality. Please watch the video if you are not familiar: [video link](https://twitter.com/wbfo/status/1268712530358292484). This should speak for itself in justifying the immediate need to defund the BPD.\n\nSincerely,\n\n[your name]"
```
Comment (Ceshion) [inline _emails/buffalo.md:30]: per conversation on #69 , please enter placeholders ([your name], etc) in all caps (i.e. [YOUR NAME]) and enter placeholders for name, address, email, and phone number in the signature

File: _emails/buffalo.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: Buffalo, NY
+permalink: "/buffalo"
+name: Buffalo, NY Resident for Defunding the BPD
+state: NY
+city: Buffalo
+layout: email
+recipients:
+  - mayor@city-buffalo.com
```
Comment (Ceshion) [inline _emails/buffalo.md:9]: please ensure email addresses here are added with no whitespace before the dash

**Comment by ceebs** [comment]
Edits have been made. Thanks @Ceshion

**Comment by teddywilson** [comment]
so sorry, i had to do a force push to gh-pages :^(  would you mind merging that branch back in so I can review?

**Comment by teddywilson** [comment]
Closing this out as buffalo was already merged. feel free to make any changes to that template as it stands right now!

**Pull request closed without merging**